### ISSUE TITLE
Is it possible to get the number of followers that an account has?

### ISSUE THREAD

**Issue by nmills3** [sirkris/Reddit.NET#167]
I'm trying to get the follower count of the user that the bot is running as, but i can't seem to find a way to do it

**Comment by sirkris** [comment]
Yes, though it's kinda hard to find because the Reddit API hides it in a weird place.  You're familiar with user subreddits, right?  What you need to do is actually retrieve the data for that user's subreddit.  The "Subscribers" property in this case is actually the user's follower count.

For example, here's a quick and dirty way to retrieve my follower count in C#:

`Console.WriteLine($"KrisCraig has {reddit.Models.Subreddits.About("u_KrisCraig").Subscribers} followers.");`

Now that I think of it, this is downright counter-intuitive.  I think maybe I should go ahead and add a "Followers" property that performs the necessary API call and retrieves/caches the subscribers count.
